StyleTwitter.css

```diff
@@ -0,0 +1,630 @@
+}
+
+.user-icon-retweet{
+    padding-right: 20px;
+}
+
+.retweeted-post > .user-icon-retweet > svg + svg{
+    height: 40px;
+    display: block;
+}
+
+.retweeted-post > .user-icon-retweet > svg:nth-child(1){
+    height: 16px;
+    display: block;
+    fill: rgb(83, 100, 113);
+    margin-left: 24px;
+}
+
+.post > .user-icon > svg{
+    height: 40px;
+    
+}
+
+.retweeted-post > .user-icon > svg{
+    height: 40px;
+    
+}
+
+.post-contents > p{
+    display: inline-block;
+    position: relative;
+    top: 8px;
+}
+
+.post-heading{
+    display: grid;
+    grid-template-rows: 1fr;
+    grid-template-columns: auto 1fr;
+    place-items: end;
+}
+
+.post-heading > a:hover{
+    text-decoration: underline;
+}
+
+.post-heading-retweet > a:hover{
+    text-decoration: underline;
+}
+
+
+.post-heading-retweet{
+    display: grid;
+    grid-template-rows: auto 1fr;
+    grid-template-columns: auto 1fr;
+    place-items: end;
+
+}
+
+.post-heading > a{
+    text-decoration: none;
+    color: black;
+}
+
+.post-heading-retweet > a{
+    text-decoration: none;
+    color: black;
+}
+
+.retweet-heading{
+    width: fit-content;
+    color: rgb(83, 100, 113);
+    margin-right: 47px;
+}
+
+.post-heading > a > h4 > span{
+    font-weight: lighter;
+    color: rgb(82, 82, 187);
+}
+
+.post-heading-retweet > a > h4 > span{
+    font-weight: lighter;
+    color: rgb(82, 82, 187);
+}
+
+
+
+.post-heading > a > svg{
+    width: 20px;
+    height: 15px;
+    
+}
+
+.post-heading-retweet > a > svg{
+    width: 20px;
+    height: 15px;
+    
+}
+
+.post-contents > h4{
+    margin-bottom: 15px;
+}
+
+.post-options{
+    display: grid;
+    grid-template-columns: repeat(5, 1fr);
+    margin-top: 30px;
+    
+}
+
+.post-options > a > svg{
+  height: 20px;  
+
+}
+
+.post-options > a > p{
+    display: inline-block;
+    position: relative;
+    bottom: 5px;
+   
+}
+
+.post-options > a{
+    text-decoration: none;
+    color: black;
+    width: fit-content;
+}
+
+.post-options > a:hover{
+    color: rgb(29, 155, 240);
+    fill: rgb(29, 155, 240);
+
+}
+
+.post-options > a:nth-child(2):hover{
+    color: rgb(1, 172, 1);
+    fill: rgb(1, 172, 1);
+
+}
+
+.post-options > a:nth-child(3):hover{
+    color: rgb(255, 73, 73);
+    fill: rgb(255, 73, 73);
+
+}
+
+
+#right-bar{
+    display: grid;
+    grid-area: right;
+    border: 2px solid rgb(228, 228, 228) ;
+    background: white;
+    position: relative;
+    width: fit-content;
+    grid-auto-rows: max-content;
+    gap: 5px;
+}
+
+#whats-happening{
+    padding: 20px 50px;
+    border-radius: 20px;
+    background-color: rgba(29, 156, 240, 0.185);
+    text-decoration: underline;
+}
+
+#who-to-follow{
+    padding: 20px 50px;
+    border-radius: 20px;
+    background-color: rgba(29, 156, 240, 0.185);
+    text-decoration: underline;
+}
+
+#right-bar > input {
+  width: 100%;
+  padding: 20px 60px 20px 20px;
+  display: block;
+  border-radius: 50px;
+  border: 2px solid rgb(228, 228, 228);
+  font-size: 17px;
+}
```

Comment:
Could add in an input:active part here to remove the blue outline when the search box is selected.